### PR TITLE
Setup:  Hello Microverse Project

### PR DESCRIPTION
**Pull request by Debas-31**
## setup: Hello Microverse Project
In this project :
- Added github workflow
- Added linters setup
- Added .gitignore file
- Added Descriptive README file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    background: blueviolet;
+    overflow: hidden;
+}
+
+h1{
+    text-align: center;
+    padding: 20rem;
+    color: #fff;
+    font-size: 3rem;
+}
```
Comment (Emmyn5600) [inline style.css:20]: - [ ] Kindly, follow the Microverse [template](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up the linters, it seems like you did the mistake while configuring it.

- [ ] Add .gitignore to keep the `test.md file and node_modules`.

- [ ] Please change the screenshot image.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Debas-31 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  
     
     * #### Make a descriptive PR with a short summary of what you implemented in the project using the past tense. For example use `Added index.html file` not `Add html file`.
     
     * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by icebox827 — CHANGES_REQUESTED** [review]
```markdown
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] **Required:** Please create a descriptive Pull Request with a summary. Add to your PR a title and a summary of what has been done in that PR.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by icebox827 — APPROVED** [review]
Hi Debas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**